parcels/examples/example_globcurrent.py

```diff
@@ -225,3 +225,22 @@ def DeleteParticle(particle, fieldset, time):
                   dt=delta(minutes=5))
 
     assert np.allclose([pset0[-1].lon, pset0[-1].lat], [pset1[-1].lon, pset1[-1].lat])
+
+
+@pytest.mark.parametrize('mode', ['scipy', 'jit'])
+@pytest.mark.parametrize('dt', [-300, 300])
+def test_globcurrent_pset_fromfile(mode, dt, tmpdir):
+    filename = tmpdir.join("pset_fromparticlefile.nc")
```

Comment:
About this `tmpdir.join`, should we make this more systematic in the example_* files? After running `py.test example_*` on my laptop, I end with many *.nc files

Reply:
Hmm, that's strange because the whole point of the `tmpdir` is that `pytest` handles this to be a temporary directory (http://doc.pytest.org/en/latest/tmpdir.html). That works for me

Reply:
Yes sure, that's the good approach. What I mean is that other examples don't use this tmpdir. We could do it for all example_* files !

Reply:
Sure, feel free to go through and check them. Should probably be a different PR though?

Reply:
Yes, I would also say a different PR